python/ray/workflow/tests/test_dag_to_workflow.py

```diff
@@ -134,6 +134,28 @@ def h():
     ray.get(dag.execute())
 
 
+def test_workflow_continuation(workflow_start_regular_shared):
+    """Test unified behavior of returning continuation inside
+    workflow and default Ray execution engine."""
+
+    @ray.remote
+    def h(a, b):
+        return a + b
+
+    @ray.remote
+    def g(x):
+        return workflow.continuation(h.bind(42, x))
+
+    @ray.remote
+    def f():
+        with InputNode() as dag_input:
+            return workflow.continuation(g.bind(dag_input.x), x=1)
```

Comment:
Oh I see. IMO, we shouldn't support need to support this. Is there a concrete use case for continuations with input?

Reply:
I'm ok with just the DAG. Additional arguments should be rare case. We can always patch it later when needed.